new_project/examples/separate_apps/src/maker.ts

```diff
@@ -0,0 +1,84 @@
+import { formatEther } from "ethers/utils";
+import readLineSync from "readline-sync";
+import { toBitcoin } from "satoshi-bitcoin-ts";
+import { checkEnvFile, startClient } from "./lib";
+
+(async function main() {
+    checkEnvFile(process.env.DOTENV_CONFIG_PATH!);
+
+    const maker = await startClient(0);
+
+    console.log(
+        "[Maker] Bitcoin balance: %f. Ether balance: %f",
+        parseFloat(await maker.bitcoinWallet.getBalance()).toFixed(2),
+        parseFloat(
+            formatEther(await maker.ethereumWallet.getBalance())
+        ).toFixed(2)
+    );
+
+    const peerId = await maker.comitClient.getPeerId();
+    const addressHint = await maker.comitClient
+        .getPeerListenAddresses()
+        .then(addresses => addresses[0]);
+
+    console.log("[Maker] peerId:", peerId);
+    console.log("[Maker] addressHint:", addressHint);
+
+    // Note that we assume the maker published an offer
+    // somehow somewhere and that someone (a taker)
+    // will take such offer by send a SWAP request to the
+    // maker's COMIT node. This initial phase is not done
+    // as part of this example.
+
+    console.log("Waiting for someone to taker my offer...");
+
+    let makerSwapHandle;
```

Comment:
:wrench: Since we only have one swap handle per file now, we can just call them `swapHandle`.